index.rst

```diff
@@ -1,57 +1,20 @@
-.. RelEng Docs documentation master file, created by
```

Comment:
This page ends up very lengthy, but I really, really, like how scannable, searchable it is. It's now much easier to get a broad overview of what's in here.